### PR TITLE
open unlock modal in publishing the template and few lint fixes

### PR DESCRIPTION
**Pull request by vikas-cis**
Open Unlock modal while publishing the template on sell and also on add item.

### CODE REVIEW

**Comment by HenkSwardt** [comment]
https://github.com/particl/particl-desktop/issues/934

File: src/app/core/market/api/cart/cart.model.ts [hunk 1]
```diff
@@ -39,8 +39,8 @@ export class Cart {
     }
 
     setTotal(): void {
-      let total: number = 0.0;
-      total += (this.subTotal.getAmount() 
+      let total = 0.0;
+      total += (this.subTotal.getAmount()
               + this.shippingTotal.getAmount()
               + this.escrowTotal.getAmount());
       this.total = new Amount(total);
```
Comment (rynomster) [inline src/app/core/market/api/cart/cart.model.ts]:
```
this.total = new Amount(this.subTotal.getAmount()
               + this.shippingTotal.getAmount()
               + this.escrowTotal.getAmount())
```

File: src/app/market/sell/add-item/add-item.component.ts [hunk 2]
```diff
@@ -232,7 +238,11 @@ export class AddItemComponent implements OnInit, OnDestroy {
         this.escrow.add(template.id, EscrowType.MAD).subscribe(
           success => {
             this.snackbar.open('Succesfully added escrow!')
-            resolve(template.id);
+            observer.next(template.id);
+            observer.complete()
+          },
+          error => {
```
Comment (rynomster) [inline src/app/market/sell/add-item/add-item.component.ts]: dont need to create a new function to call an existing function
just use 
```
success => {.....
},
observer.error);
```

**Comment by vikas-cis** [comment]
@rynomster I have resolve the conflictions but I was wondering regarding the call back function as removing the cart wasn't working when we called `getCart` as callback so I changed a bit in the definition of that function.

**Pull request merged**